app/models/application_choice.rb

```diff
@@ -64,17 +64,23 @@ def recruitment_cycle
   end
 
   def days_left_to_respond
-    if respond_to?(:pg_days_left_to_respond)
-      # pre-computed by sorting query
-      return pg_days_left_to_respond
-    end
-
-    if status == 'awaiting_provider_decision'
+    # if respond_to?(:pg_days_left_to_respond)
+    #   # pre-computed by sorting query
+    #   return pg_days_left_to_respond
+    # end
```

Comment:
can we remove the commented out code please.

Reply:
👍